### PR TITLE
 Adding files

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ansible
```
Comment (rohitthakur2590) [inline requirements.txt:1]: this whole file could be deleted

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+[testenv:black]
+install_command = pip install {opts} {packages}
+commands =
+  black -v -l79 {toxinidir}
+
+[testenv:linters]
+install_command = pip install {opts} {packages}
+commands =
+  black -v -l79 --check {toxinidir}
+  flake8 {posargs}
```
Comment (rohitthakur2590) [inline tox.ini:19]: - we don't have any code so we can remove black
- we can remove flake as well as we don't have any python

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+commands = {posargs}
+
+[flake8]
+# E123, E125 skipped as they are invalid PEP-8.
+
+show-source = True
+ignore = E123,E125,E402,W503
+max-line-length = 160
+builtins = _
+exclude = .git,.tox,tests/unit/compat/
```
Comment (rohitthakur2590) [inline tox.ini:32]: not required

**Review by rohitthakur2590 — CHANGES_REQUESTED** [review]

**Comment by GomathiselviS** [comment]
recheck

File: bindep.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# This is a cross-platform list tracking distribution packages needed by tests;
```
Comment (pabelanger) [inline bindep.txt:1]: I don't think we need this file

Comment (rohitthakur2590) [reply bindep.txt:1]: ++

File: plugins/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Collections Plugins Directory
```
[the remark this answers is not in the document]
Comment (rohitthakur2590) [reply plugins/README.md:1]: @GomathiselviS  Yes, I think it better to remove this. IMO we are good with the existing README.md

**Review by rohitthakur2590 — APPROVED** [review]

File: tox.ini [hunk 6]
```diff
@@ -0,0 +1,16 @@
+[tox]
+minversion = 1.4.2
+envlist = linters
+skipsdist = True
+
+[testenv]
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+
+[testenv:linters]
```
Comment (pabelanger) [inline tox.ini:10]: please add tox-linters jobs to ansible.network collection

**Pull request merged**